eland/ml/transformers/xgboost.py

```diff
@@ -107,6 +107,7 @@ def build_tree_node(self, row: pd.Series, curr_tree: int) -> TreeNode:
                 decision_type=self._node_decision_type,
                 left_child=self.extract_node_id(row["Yes"], curr_tree),
                 right_child=self.extract_node_id(row["No"], curr_tree),
+                default_left=row["Yes"] == row["Missing"],
```

Comment:
I don't know how the tree works internally. What does this line mean - we're traversing the left subtree if the "Yes" value is the same as the "Missing" value? Could you summarize what that means?

Reply:
Each node of the tree (except leafs that own value) have 2 children nodes: left and right.
Based on the value of the feature we decide to go left or right. If the feature is missing we can decide (default_left) to go to the left or to the right (training decide for us).

When exporting xgboost the tree is represented as a table and:
- `row['Yes']` contains the id of the left node
- `row['No']` contains the id of the right node
- `row['Missing']` contains the id of the node when a value is missing and is always equals to `row['Yes']` or `row['No']`

So this code convert this xgboost specific structure into one that ES understand.

Reply:
Gotcha, thanks for the clarification!